### PR TITLE
Adding rollbar (#115)

### PR DESCRIPTION
**Pull request by urfolomeus**
* Tidy up Gemfile

* Locking down gem versions

* Adding Rollbar

* Adding rollbar error tracking to the blog feed fetcher

* Adding rollbar error tracking to the all feed fetcher

* Adding rollbar error tracking to the unpublished flagger

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,54 +2,56 @@ source 'https://rubygems.org'
...
 end
 
 group :production do
-  gem 'unicorn'
-  gem 'therubyracer'
-  gem 'mailjet'
+  gem 'mailjet',                    '~> 1.0.3'
+  gem 'therubyracer',               '~> 0.12.2'
+  gem 'unicorn',                    '~> 4.9.0'
 end
```
Comment (tomas-stefano) [inline Gemfile]: Is this indentation necessary?

**Comment by Tr4pSt3R** [comment]
it's all good. 👍 👍

**Comment by tomas-stefano** [comment]
After the code format, lgtm 👍

File: Gemfile (hunk 1 above)
Comment (urfolomeus) [inline Gemfile]: no

**Comment by urfolomeus** [comment]
Thanks @Tr4pSt3R @tomas-stefano

**Pull request merged**